MmSupervisorPkg/Core/aux_config.spam.cfg

```diff
@@ -1,5 +1,17 @@
 autogen = true
 
+[[key]]
+signature = 0x46504F4C

```

Comment:
@Javagedes I think the way this signature is being accepted is a bit counter-intuitive as the definition of these signatures are normally formatted as 4 characters. Can we try to update the tool to see if any other options?

Reply:
Handled by https://github.com/microsoft/mu_feature_mm_supv/pull/267.